src/kademlia/routing_table.cpp

```diff
@@ -227,22 +220,22 @@ int routing_table::depth() const
 
 	// maybe the table is deeper now?
 	while (m_depth < int(m_buckets.size())-1
-		&& int(m_buckets[m_depth+1].live_nodes.size()) >= m_bucket_size / 2)
+		&& int(m_buckets[m_depth + 1].live_nodes.size()) >= m_bucket_size / 2)
 	{
 		++m_depth;
 	}
 
 	// maybe the table is more shallow now?
 	while (m_depth > 0
-		&& int(m_buckets[m_depth-1].live_nodes.size()) < m_bucket_size / 2)
+		&& int(m_buckets[m_depth - 1].live_nodes.size()) < m_bucket_size / 2)
 	{
 		--m_depth;
 	}
 
 	return m_depth;
 }
 
-#ifndef TORRENT_DISABLE_LOGGING
+#if TORRENT_USE_IOSTREAM
 void routing_table::print_state(std::ostream& os) const
```

Comment:
I'm not sure exactly where this is used, but I think, historically, it's been used as a debug logging facility. I'm a bit tempted to weed out uses of the ``TORRENT_USE_IOSTREAM`` macro.